velox/expression/tests/ExprTest.cpp

```diff
@@ -612,13 +602,6 @@ TEST_P(ParameterizedExprTest, constantFolding) {
     auto typedExpr = parseExpression("codepoint('abcdef')", ROW({}));
     EXPECT_NO_THROW(exec::ExprSet exprSet({typedExpr}, execCtx_.get(), true));
   }
-
-  {
-    registerFunction<NoArgFunction, int64_t>({"no_arg_func"});
```

Comment:
Why was this test deleted?

Reply:
We don't want to introduce breaking changes in this PR. Hence, we'd like to keep the existing functionality. I don't understand the concern about constant folding functions with no arguments. Any function (w/ or w/o args) can be non-deterministic. We have metadata about that and tryConstantFold handles this properly, I believe.